index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8" />
+<title>The Code Magazine</title>
+<style>
+    body {
+        margin-left: 20px;
+    }
+</style>
+</head>
+<body>
+    <h1>📘 The Code Magazine</h1>
+
+    <h2>The Basic Language of the Web: HTML</h2>
+    <img src="D://Tekwill/Tekwill-Begin/img/laura-jones.jpg" alt="Laura Jones" width="60px" height="60px">
```

Comment:
Since you already have the images inside your project, you should use relative path instead of absolute

Reply:
I had a problem, have used */* to show the relative path, but it was incorrect and images didn't work. Right now, i wrote the path without */* and it's work. (stupid mistake :) )
![image](https://github.com/DirCd/Tekwill-Begin/assets/141740084/21dd0541-b1ac-4f7b-9fa5-99dd38725782)